flopy/modflowusg/mfusgbcf.py

```diff
@@ -0,0 +1,675 @@
+    hy : float or array of floats (nlay, nrow, ncol)
+        hydraulic conductivity (only read if laycon is 1 or 3)
+        (default is 1.0)
+    vcont : float or array of floats (nlay-1, nrow, ncol)
+        vertical leakance between layers (default is 1.0)
+    kv : float or array of floats (nlay, nrow, ncol)
+        is the vertical hydraulic conductivity of the cell and the leakance is
+        computed for each vertical connection.
+    sf1 : float or array of floats (nlay, nrow, ncol)
+        specific storage (confined) or storage coefficient (unconfined),
+        read when there is at least one transient stress period.
+        (default is 1e-5)
+    sf2 : float or array of floats (nlay, nrow, ncol)
+        specific yield, only read when laycon is 2 or 3 and there is at least
+        one transient stress period (default is 0.15)
+    wetdry : float
+        a combination of the wetting threshold and a flag to indicate which
+        neighboring cells can cause a cell to become wet (default is -0.01)
+    ksat : float or array of floats (njag)
+        inter-block saturated hydraulic conductivity or transmissivity
+        (if IKCFLAG = 1) or the inter-block conductance (if IKCFLAG = - 1)
+        of the connection between nodes n and m.
+    extension : string
+        Filename extension (default is 'bcf')
+    unitnumber : int
+        File unit number (default is None).
+    filenames : str or list of str
+        Filenames to use for the package and the output files. If
+        filenames=None the package name will be created using the model name
+        and package extension and the cbc output name will be created using
+        the model name and .cbc extension (for example, modflowtest.cbc),
+        if ipakcbc is a number greater than zero. If a single string is passed
+        the package will be set to the string and cbc output name will be
+        created using the model name and .cbc extension, if ipakcbc is a
+        number greater than zero. To define the names for all package files
+        (input and output) the length of the list of strings should be 2.
+        Default is None.
+    add_package : bool
+        Flag to add the initialised package object to the parent model object.
+        Default is True.
+
+    Methods
+    -------
+
+    See Also
+    --------
+
+    Notes
+    -----
+
+    Examples
+    --------
+
+    >>> import flopy
+    >>> ml = flopy.modflowusg.ModflowUsg()
+    >>> disu = flopy.modflowusg.ModflowUsgDisU(model=ml, nlay=1, nodes=1,
+                 iac=[1], njag=1,ja=np.array([0]), fahl=[1.0], cl12=[1.0])
+    >>> bcf = flopy.modflowusg.ModflowUsgBcf(ml)"""
+
+    def __init__(
+        self,
+        model,
+        **kwargs,
+    ):
+        """
+        Constructs the ModflowUsgBcf object.
+
+        Overrides the parent ModflowBcf object."""
+        msg = (
+            "Model object must be of type flopy.modflowusg.ModflowUsg\n"
+            + "but received type: {type(model)}."
```

Comment:
```suggestion
            f"but received type: {type(model)}."
```
this same edit applies 14 other times in this PR